.gitignore

```diff
@@ -9,3 +9,5 @@ examples/Outputs/
 JuliaSysimage.dll
 
 docs/build/
+input/
+output/
```

Comment:
Can we move the running of the model outside of the package repo directory?

Reply:
Also, should we include the newline at end of file?

Reply:
You can mount the input/output folders where ever you want on the host system. That's a user decision but would be common to mount in this location.